Added Double column Resume Template

SauravGitte opened the pull request:
-Renamed the 'Resume' folder to 'Resume-single-col' .
-Created 'Resume-double-col' folder and added the Template .

SauravGitte committed:
Added Double column Resume

SauravGitte committed:
Updated Readme

atharva-gadekar reviewed (changes requested):
Can you keep the Resume folder intact and create two subfolders inside it for single and double col?

Also, can you group [Final Semester Thesis.zip](https://github.com/atharva-gadekar/Overleaf-Templates-IIITA/blob/main/Final%20Semester%20Thesis.zip) and [MobileVIT Final Thesis.pdf](https://github.com/atharva-gadekar/Overleaf-Templates-IIITA/blob/main/MobileVIT%20Final%20Thesis.pdf) into a single folder named `Major_Project_Thesis`

SauravGitte committed:
Re-structured the folders

SauravGitte commented:
Hello Atharv sir,
I've made the requested changes as per your feedback:

- Kept the Resume folder intact and added single_col and double_col subfolders inside it .
- Moved both the thesis files into a new folder named Major_Project_Thesis .

Please let me know if there’s anything else to update.

atharva-gadekar reviewed (approved):
LGTM
